Formula/man-db.rb

```diff
@@ -21,32 +22,17 @@ class ManDb < Formula
 
   depends_on "pkg-config" => :build
   depends_on "groff"
+  depends_on "libpipeline"
 
   uses_from_macos "zlib"
 
   on_linux do
     depends_on "gdbm"
   end
 
-  resource "libpipeline" do
-    url "https://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.5.3.tar.gz"
-    sha256 "5dbf08faf50fad853754293e57fd4e6c69bb8e486f176596d682c67e02a0adb0"
-  end
-
   def install
-    resource("libpipeline").stage do
-      system "./configure",
-        "--disable-dependency-tracking",
-        "--disable-silent-rules",
-        "--prefix=#{buildpath}/libpipeline",
-        "--enable-static",
-        "--disable-shared"
-      system "make"
-      system "make", "install"
-    end
-
-    ENV["libpipeline_CFLAGS"] = "-I#{buildpath}/libpipeline/include"
-    ENV["libpipeline_LIBS"] = "-L#{buildpath}/libpipeline/lib -lpipeline"
+    ENV["libpipeline_CFLAGS"] = "-I#{Formula["libpipeline"].opt_include}"
+    ENV["libpipeline_LIBS"] = "-L#{Formula["libpipeline"].opt_lib} -lpipeline"
```

Comment:
It might be that you don't need these anymore, actually. I think it was only needed previously because of the non-standard install location.

Reply:
Yes, I think you are right.